src/main/java/controller/StudentsOperationsClassic.java

```diff
@@ -0,0 +1,73 @@
+package controller;
+
+import model.Student;
+
+import java.util.*;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/2/2019.
+ */
+public class StudentsOperationsClassic {
+
+    private static List<Student> students = StudentsList.students;
```

Comment:
unnecessary reference students to the static object

Reply:
done - removed logic of static methods. The class was removed as redundant